src/julia.h

```diff
@@ -1958,6 +1948,13 @@ typedef struct _jl_task_t {
     size_t bufsz; // actual sizeof stkbuf
     unsigned int copy_stack:31; // sizeof stack for copybuf
     unsigned int started:1;
```

Comment:
Can we fold `started` into the `reentrant_timing` field? Otherwise started and copystack don't have a known location in the struct, which makes identifying padding bytes harder.